src/compiler-debugging.md

```diff
@@ -7,6 +7,21 @@ advice about specific parts of the compiler (e.g. the [Queries Debugging and
 Testing chapter](./incrcomp-debugging.html) or the [LLVM Debugging
 chapter](./backend/debugging.md)).
 
+## Configuring the compiler
+
+By default, rustc is built without most debug information. To enable debug info, 
+set `debug = true` in your config.toml. 
```

Comment:
```suggestion
set `debug = true` in your `config.toml`.
```

Reply:
It seems config.toml is not in backticks anhywhere else in the file. I'll remove them from `config.toml.example`